omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -250,6 +315,12 @@ public void setModelAttributes(PageModel model, Account account, OtherAccountDat
 				force = (user.getUserId() == null) ? true : account.isSupposedToChangePassword(user);
 			}
 			userProperties.put(OpenmrsConstants.USER_PROPERTY_CHANGE_PASSWORD, force);
```

Comment:
Same here as above.

Reply:
I think you have mentioned the next line. Changed